web/pdf_page_view.js

```diff
@@ -452,11 +452,41 @@ class PDFPageView {
 
     let resultPromise = paintTask.promise.then(function() {
       return finishPaintTask(null).then(function () {
+        let pump;
```

Comment:
I really don't understand any of these changes; what's the point of streaming if you're still going to fetch all of the textContent here before rendering the textLayer?

As far as can tell, isn't the point that you just want to pass a `readableStream` via `web/text_layer_builder.js` onto `src/display/text_layer.js` where it's then used to actually get the textContent!? 

I.e. shouldn't this all of this code just be replaced with:
```javascript
if (textLayer) {
  pdfPage.streamTextContent({
    normalizeWhitespace: true,
  }).then(function(readableStream) {
    textLayer.setTextContentStream(readableStream);
    textLayer.render(TEXT_LAYER_RENDER_DELAY);
  });
}
```